app/templates/public.hbs

```diff
@@ -35,9 +35,9 @@
             {{/if}}
           {{/if}}
         {{else}}
-          {{#ui-sticky context='#public-event-content' observeChanges=true}}
+          <div class="ui fluid" style="position:sticky; top:0px;">
```

Comment:
I don't think top 0px is right. Besides, it should be moved to CSS file

Reply:
it's working on top:0px -  https://youtu.be/8cqPH3b4X58
and if i should change then what should be the value.